presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/APIFileHandler.java

```diff
@@ -64,62 +68,73 @@ public Response getFile(String path)
         }
         catch (FileNotFoundException e) {
             LOGGER.error(e, "File '%s' not found", path);
-            return Response.status(Status.NOT_FOUND).build();
+            return Response.status(Status.NOT_FOUND)
+                    .entity("File not found").build();
         }
         catch (IOException e) {
             LOGGER.error(e, "Failed to process file '%s'", path);
-            return Response.status(Status.INTERNAL_SERVER_ERROR).build();
+            return Response.status(Status.INTERNAL_SERVER_ERROR)
+                    .entity("Fail to process file").build();
         }
     }
 
     public Response replaceFileFromURL(String path, String url)
     {
         try {
             AgentFileUtils.replaceFile(Paths.get(baseDir.toString(), path), url);
-            LOGGER.debug("Successfully replaced file '%s' with url '%s'", path, url);
-            return Response.status(Status.ACCEPTED).build();
+            LOGGER.debug("Replacing file '%s' with url '%s' in progress", path, url);
+            return Response.status(Status.ACCEPTED)
+                    .entity("Replacing file with url in progress").build();
```

Comment:
This operation is not asynchronous; we should be sending back an `OK` with a message that the file was replaced.